doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -807,8 +807,13 @@ Modem libraries
 
 * :ref:`nrf_modem_lib_readme`:
 
-  * Added the Kconfig option  :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_CHUNK_SZ` to process traces in chunks.
-    This can improve the availability of trace memory, and thus reduce the chances of losing traces.
+
+  * Added
+
+    * The Kconfig option  :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_CHUNK_SZ` to process traces in chunks.
+    * The :kconfig:option:`CONFIG_NRF_MODEM_LIB_FAULT_THREAD_STACK_SIZE` Kconfig option to allow the application to set the modem fault thread stack size.
+      This can improve the availability of trace memory, and thus reduce the chances of losing traces.
+
```

Comment:
```Suggestion
  * Added

    * The Kconfig option  :kconfig:option:`CONFIG_NRF_MODEM_LIB_TRACE_BACKEND_UART_CHUNK_SZ` to process traces in chunks.
      This can improve the availability of trace memory, and thus reduce the chances of losing traces.
    * The :kconfig:option:`CONFIG_NRF_MODEM_LIB_FAULT_THREAD_STACK_SIZE` Kconfig option to allow the application to set the modem fault thread stack size.
```